docs/sources/open-source.md

```diff
@@ -133,38 +155,56 @@ lt --port 8080 -s pretty-turkey-83 --print-requests
     socket_mode_enabled: false
   ```
 
-6. Go to your "OnCall" -> "Env Variables" and set:
+6. Set environment variables by navigating to your Grafana OnCall, then click **Env Variables** and set the following:
     ```
     SLACK_CLIENT_OAUTH_ID = Basic Information -> App Credentials -> Client ID
     SLACK_CLIENT_OAUTH_SECRET = Basic Information -> App Credentials -> Client Secret
     SLACK_SIGNING_SECRET = Basic Information -> App Credentials -> Signing Secret
     SLACK_INSTALL_RETURN_REDIRECT_HOST = << OnCall external URL >>
     ```
 
-7. Go to "OnCall" -> "ChatOps" -> "Slack" and install Slack Integration
+7. From OnCall, navigate to **ChatOps**, select Slack and click **Install Slack integration**.
+
+8. Configure additional Slack settings as needed and your integration is ready for use.
+
+## Telegram integration for Grafana OnCall OSS
+
+The Telegram integration for Grafana OnCall is designed for collaborative team work and improved incident response.
+### Before your begin
+
+Review the following requirements to configure Telegram for Grafana OnCall OSS:
+- Ensure your Grafana OnCall environment is running.
+- Grafana OnCall must be externally available and provide HTTPS endpoint to establish subscription on Telegram events.
+- Create new Telegram Group and private Telegram channel for alerts.
+
+### Configure Telegram for Grafana OnCall OSS
+
+1. Ensure your OnCall environment is up and running.
+
+2. Request [BotFather](https://t.me/BotFather) for a key, then add your key in `TELEGRAM_TOKEN` in your Grafana OnCall **Env Variables**.
 
-8. All set!
+3. Set `TELEGRAM_WEBHOOK_HOST` with your external URL for your Grafana OnCall. 
 
-## Telegram Setup
+4. From the **ChatOps** tab in Grafana OnCall, click **Telegram** and your integration is now ready for use.
 
-- Telegram integrations requires OnCall to be externally available and provide https endpoint. 
-- Telegram integration in OnCall is designed for collaborative team work. It requires Telegram Group and a Telegram Channel (private) for alerts. 
+## Connect Grafana Cloud to Grafana OnCall OSS
 
-1. Make sure your OnCall is up and running.
+Open source Grafana OnCall can be connected to Grafana Cloud to configure a variety of notifications.
 
-2. Respectfully ask [BotFather](https://t.me/BotFather) for a key, put it in `TELEGRAM_TOKEN` in "OnCall" -> "Env Variables".
+The benefits of connecting to Grafana Cloud include:
+- Heartbeat notification
+- SMS for user notifications
+- Phone calls for user notifications.
 
-3. Set `TELEGRAM_WEBHOOK_HOST` with your external url for OnCall. 
+To connect to Grafana Cloud, refer to the **Cloud** page in your OSS Grafana OnCall instance. 
```

Comment:
**Cloud** page? Is this page only available in OSS?

Reply:
that was my interpretation but not confirmed